contracts/v4/Registration.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {Core} from "./Core.sol";
+
+contract Registration {
```

Comment:
this is outdated, can you please have a look at https://github.com/immutable/contracts/tree/main/contracts/bridge/x/v4